env/setup.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+root=$( dirname $( wp config path ) )
+
+wp db import "${root}/env/bpmain_bp_xprofile_data.sql"
+
+wp theme activate wporg-5ftf-2024
+wp plugin activate wporg-5ftf
+
+wp rewrite structure '/%postname%/'
+wp rewrite flush
```

Comment:
I still had to go to Permalinks to re-save this, maybe it would work at the end of this script?

Reply:
It may. I've seen this behavior on other project as well and nothing appears to flush it properly.